calendar.groovy

```diff
@@ -0,0 +1,64 @@
+@Grab(group = 'org.mnode.ical4j', module = 'ical4j', version = '1.0.6')
+import net.fortuna.ical4j.util.*
+import net.fortuna.ical4j.data.*
+import net.fortuna.ical4j.model.*
+import net.fortuna.ical4j.model.property.*
+import net.fortuna.ical4j.model.parameter.*
+import net.fortuna.ical4j.model.component.*
+
+import static java.util.Objects.requireNonNull
+
+def arg(name, desc) {
+    def value = System.getProperty(name)
+    requireNonNull(value, "Please specify -D${name} arg - ${desc}")
+}
+
+eventDateTime = arg("dateTime", "DateTime in `d/M/yyyy H:m` format")
+outfile = arg("out", "Calendar output file name")
+
+// Obtains local timezone object (in our case Europe/Riga)
+TimeZoneRegistry registry = TimeZoneRegistryFactory.getInstance().createRegistry()
+TimeZone timezone = registry.getTimeZone("Europe/Riga")
+
+// Calculates event start/end datetime (from command line arguments)
+java.util.Calendar startDate = new GregorianCalendar()
+startDate.setTime(new Date().parse("d/M/yyyy H:m", eventDateTime, timezone))
+java.util.Calendar endDate = new GregorianCalendar()
+endDate.setTime(startDate.getTime())
+endDate.add(java.util.Calendar.MINUTE, 30)
+
+// Create the event
+String eventName = "Latcraft On Air"
+DateTime start = new DateTime(startDate.getTime())
+DateTime end = new DateTime(endDate.getTime())
+VEvent meeting = new VEvent(start, end, eventName)
+// Add timezone info
+VTimeZone tz = timezone.getVTimeZone()
+meeting.getProperties().add(tz.getTimeZoneId())
+// Add unique identifier
+UidGenerator ug = new UidGenerator("latcraft")
+meeting.getProperties().add(ug.generateUid())
+// Add event location details
+meeting.getProperties().add(new LocationType("Youtube"))
+meeting.getProperties().add(new Location("https://www.youtube.com/channel/UCvzMZyJZZ3XYQwbvOACVYrQ"))
+meeting.getProperties().add(new Url(new ParameterList(), "https://www.youtube.com/channel/UCvzMZyJZZ3XYQwbvOACVYrQ"))
+// Add description
+meeting.getProperties().add(new Description("For more videos subscribe to Latcraft Youtube channel: https://www.youtube.com/channel/UCvzMZyJZZ3XYQwbvOACVYrQ"))
+// Add event organizer details
+ParameterList pl = new ParameterList();
+pl.add(new Cn("Latcraft"))                              // Visible name of organizer
+pl.add(new SentBy("hello@latcraft.lv"))                 // Not important
+def orgEmail = URI.create("mailto:hello@latcraft.lv")   // Will be used in replies and accept/reject mail notifications
```

Comment:
@eduardsi Please take a closer look. Should we use `hello@latcraft.lv` here?
At the moment the email address will be used to sent calendar event accept/reject mail notifications (e.g. it might be 200+ emails). Do you want me to specify different email address here? (e.g. `calendar@latcraft.lv` or `hello+calendar@latcraft.lv`)